src/RestController.php

```diff
@@ -327,9 +327,30 @@ public function onDispatch(MvcEvent $e)
         // Check for an API-Problem in the event
         $return = $e->getParam('api-problem', false);
 
-        // If no API-Problem, dispatch the parent event
         if (!$return) {
-            $return = parent::onDispatch($e);
+            // Override RESTful methods
+            $request = $e->getRequest();
+            $method = strtolower($request->getMethod());
+
+            switch ($method) {
+                case 'delete':
+                    $routeMatch = $e->getRouteMatch();
+                    $id = $this->getIdentifier($routeMatch, $request);
+                    $data = $this->processBodyContent($request);
+
+                    if ($id !== false) {
+                        $return = parent::onDispatch($e);
+                        break;
+                    }
```

Comment:
You could have this do a `continue` instead; that should drop to the `default` target of the `switch` statement.